src/components/util/virtualized-table.js

```diff
@@ -0,0 +1,381 @@
+/**
+ * This class has been taken from 'Virtualized Table' example at https://material-ui.com/components/tables/
```

Comment:
Is it a copy paste from study? Maybe this is the rigth time to share it in commons ?

Reply:
i'll move it in a separate PR in gridexplore and gridstudy

Reply:
No, it has to be done before this PR